### PR TITLE
updating nav header and footer for external view

### PR DESCRIPTION
**Pull request by tarafanlin**
- adding CTA in the header for sign up/log in
- rearrange other actions to the footer
- apply max-width content and full-width background
Screen Shot 2020-10-21 at 9 34 32 PM

### CODE REVIEW

File: components/core/WebsitePrototypeHeader.js [hunk 1]
```diff
@@ -1,97 +1,80 @@
...
+            css={STYLES_LINK}
+            href="/"
+            style={{ marginRight: 16, position: "relative", top: "1px" }}
+          >
+            <Logo style={{ height: 20 }} />
+          </a>
+        </div>
+        <div css={STYLES_RIGHT}>
+          <a css={STYLES_LINK} href="/_">
+            Sign up⼁Log in
```
Comment (jimmylee) [inline components/core/WebsitePrototypeHeader.js:75]: Sign up,
Sign in

are our actions

**Review by jimmylee — CHANGES_REQUESTED** [review]
Lets do a 24px gap for the bottom right items and see how that feels, if its too wide lets do 16px, same with the top we don't need the `|` seperator

**Comment by tarafanlin** [comment]
@jimmylee new changes made
Screen Shot 2020-10-21 at 10 41 52 PM

**Review by jimmylee — APPROVED** [review]
LGTM

**Pull request merged**